source/_components/garage_door.rpi_gpio.markdown

```diff
@@ -41,4 +43,6 @@ Configuration variables:
   - **name** (*Optional*): Name to use in the Frontend.
   - **relay_pin** (*Required*): The pin of your Raspberry Pi where the relay is connected.
   - **state_pin** (*Required*): The pin of your Raspberry Pi to retrieve the state.
+  - **state_pull_mode** (*Optional*): The direction the State pin is pulling. It can be UP or DOWN.
+  - **relay_time** (*Optional*): The time that the relay will be on for in seconds.
```

Comment:
Can you please add the default value? Would the users save the trouble of searching it in the source.